ui/main/src/app/services/admin/AdminProcessesService.ts

```
@@ -33,12 +33,12 @@
         return this.queryAllProcesses();
     }
     private static queryAllProcesses(): Observable<Process[]> {
-        return this.adminprocessServer.queryAllProcesses().pipe(
-            map((adminprocessResponse) => {
-                if (adminprocessResponse.status === ServerResponseStatus.OK) {
-                    return adminprocessResponse.data;
+        return this.adminProcessesServer.queryAllProcesses().pipe(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)